src/app/auth-page/auth-page.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-auth-page',
+  templateUrl: './auth-page.component.html',
+  styleUrls: ['./auth-page.component.scss']
+})
+export class AuthPageComponent implements OnInit {
+  active = 1;
```

Comment:
Don't forget to use access modifiers. "Active" flag is not needed, as stated in prev comment.
If there will be a need to define such variable, use Boolean type and a name like "isActive"/"isSignUpButtonActive"